plugins/circleci/src/route-refs.tsx

```diff
@@ -22,8 +22,8 @@ export const circleCIRouteRef = createRouteRef({
 });
 
 /** @public */
-export const circleCIBuildRouteRef = createSubRouteRef({
-  id: 'circle-ci/build',
+export const circleCIWorkflowRouteRef = createSubRouteRef({
+  id: 'circle-ci/workflow',
   parent: circleCIRouteRef,
-  path: '/:buildId',
+  path: '/workflows/:workflowId',
```

Comment:
The old URL will no longer render anything. I can't redirect to the workflow page because the IDs used in these routes are different, but I could add a redirect to the workflows list. Is the concern that there could be some custom implementations accessing the old url? In this plugin, the routes are defined internally (not added to the App routes).

Reply:
It's just that (full) URLs are meant to be stable over time when possible. People may have browser bookmarks. They may have links in their sidebar Playlist plugin. They may have generated links in other tooling like their CI pipelines.

Maybe a redirect can help

Reply:
That makes sense, thanks for explaining. I have added the redirect.